Solutions/Branwen/BranwenFileInfo.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Branwen
+{
+	public class BranwenFileInfo
+	{
+		public string Name
+		{
+			get;
+			set;
+		}

```

Comment:
remove all of these get; set; placeholders.  They are not really needed unless you are going to do something special with the getter and setter.  They are implied and put in by the compiler later.  

public string Name;